src/main/java/org/opentestsystem/ap/ims/rest/v1/ItemApi.java

```diff
@@ -88,6 +92,31 @@ public Item getItem(@PathVariable final String itemId) {
      * @param itemId The unique item identifier.
      * @return The changes associated with an item.
      */
+    @GetMapping("/{itemId}/history/csv")
+    public void getItemHistoryCsv(@PathVariable final String itemId, final HttpServletResponse response) {
+        log.debug("find item csv {}", itemId);
+
+        final String csvString = itemService.findItemHistoryCsvString(itemId);
+        InputStream csvStream = null;
+        try {
+            csvStream = IOUtils.toInputStream(csvString, "UTF-8");
```

Comment:
I wonder if you meant "response" and not "request".   To be safe we could set response.setContentType("text/csv; charset=UTF-8;")

I'll do that as I don't think it hurts.  I don't know though if setting the encoding on the stream is enough.  I will guess it is.